apps/frontend/.env.example

```
@@ -6,4 +6,5 @@ NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=#736891312580
 NEXT_PUBLIC_FIREBASE_APP_ID=#1:736891312580:web:2752bc815204c69fcbec91
 
 NEXT_PUBLIC_BACKEND_URL=#http://localhost:3000
-NEXT_PUBLIC_HASURA_GRAPHQL_URL=#http://localhost:8080/v1/graphql
\ No newline at end of file
+NEXT_PUBLIC_HASURA_GRAPHQL_URL=#http://localhost:8080/v1/graphql
+NEXT_PUBLIC_HASURA_ADMIN_SECRET=#myadminsecretkey
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)